mmpose/apis/inference.py

```diff
@@ -145,6 +146,20 @@ def _inference_single_pose_model(model,
     if device.type == 'cpu':
         device = -1
 
+    # to support multi-frame setting, the imgs_or_paths is a list/tuple
+    # even if it has only one image/path
+    assert isinstance(imgs_or_paths, list)
```

Comment:
please double-check if this will affect other demos, e.g. image demo.

Reply:
OK, I will check it.

Reply:
I have checked the input img_or_path in L377-L378, and make it a list before inputting it to `_inference_single_pose_model`, so it will not affect the previous demo usage.